js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,29 @@
+function duplicateLetters(...args) {
+  const string = args[0];
```

Comment:
Also string is not a good variable name, can be confused with string type.

Reply:
I agree it's confusing.